lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -122,6 +123,47 @@ def config_with_filter(
         return CANNOT_CONFIG
 
 
+def config_with_devices(args, mounts, current_wwids, wanted_wwids):
+
+    # Check if the lvm devices is already configured.
+    if lvmdevices.is_configured():
+        print("LVM devices already configured for vdsm")
+        return
```

Comment:
lvmdevices.is_configured() check only if use_devices is set. But to we also need to check that system.devices exists, and that is is valid. I'm not sure how we should 
check that, or we we have a good way to do this.

Lets discuss this with David.

Reply:
We can use "lvmdevices --check" here, but need to ask David about this.